addons/music/fnc_getMusicPlaying.sqf

```diff
@@ -0,0 +1,44 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicPlaying
+
+Description:
+    A function used to return the current time on playing music. Must have been started with CBA_fnc_playMusic
+    
+Parameters:
+    BOOL: if true, return bool (default: false)
+
+Returns:
+    array [class, music time, time left] OR bool (true for playing music)
```

Comment:
I'd personally prefer if there was `CBA_fnc_getMusicPlaying` reporting the array and `CBA_fnc_isMusicPlaying` always returning the boolean.